packages/amino/src/multisig.ts

```diff
@@ -0,0 +1,43 @@
+import { toHex } from "@cosmjs/encoding";
+import { Uint53 } from "@cosmjs/math";
+
+import { pubkeyToRawAddress } from "./addresses";
+import { MultisigThresholdPubkey, SinglePubkey } from "./pubkeys";
+
+/**
+ * Compare arrays lexicographically.
+ *
+ * Returns value < 0 if `a < b`.
+ * Returns value > 0 if `a > b`.
+ * Returns 0 if `a === b`.
+ */
+export function compareArrays(a: Uint8Array, b: Uint8Array): number {
+  return toHex(a).localeCompare(toHex(b));
```

Comment:
Interesting. Can we make the locale explicit?

Reply:
yeah, hex encoding preserves ordering because the ASCII value of 0-9 is smaller than the one of a-f.

Locale is a good point. I will check.